src/main/java/blacknut/ui/Blacknut.java

```diff
@@ -0,0 +1,340 @@
+//        System.out.println(" Got it. I've added this task:");
+//        System.out.println("   " + newTask);
+//        System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+//    }
+//
+//    private static void addDeadline(ArrayList<Task> tasks, String input) throws IncorrectFormatException {
+//        String[] parts = input.substring(9).split(" /by ");
+//        if (parts.length != 2 || parts[0].trim().isEmpty() || parts[1].trim().isEmpty()) {
+//            throw new IncorrectFormatException("The format for a deadline should be: deadline <description> /by <d/M/yyyy HHmm>");
+//        }
+//        Task newTask = new Deadline(parts[0].trim(), parts[1].trim());
+//        tasks.add(newTask);
+//        saveTasksToFile(tasks);
+//        System.out.println(" Got it. I've added this task:");
+//        System.out.println("   " + newTask);
+//        System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+//    }
+//
+//
+//    private static void addEvent(ArrayList<Task> tasks, String input) throws IncorrectFormatException {
+//        String[] parts = input.substring(6).split(" /from | /to ");
+//        if (parts.length != 3 || parts[0].trim().isEmpty() || parts[1].trim().isEmpty() || parts[2].trim().isEmpty()) {
+//            throw new IncorrectFormatException("The format for an event should be: event <description> /from <yyyy-mm-dd> /to <yyyy-mm-dd>");
+//        }
+//        Task newTask = new Event(parts[0].trim(), parts[1].trim(), parts[2].trim());
+//        tasks.add(newTask);
+//        saveTasksToFile(tasks);
+//        System.out.println(" Got it. I've added this task:");
+//        System.out.println("   " + newTask);
+//        System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+//    }
+//
+//    // Method to delete a task
+//    private static void deleteTask(ArrayList<Task> tasks, String input) throws InvalidTaskNumberException {
+//        try {
+//            int index = Integer.parseInt(input.split(" ")[1]) - 1;
+//            if (index >= 0 && index < tasks.size()) {
+//                Task removedTask = tasks.remove(index);
+//                saveTasksToFile(tasks);
+//                System.out.println(" Noted. I've removed this task:");
+//                System.out.println("   " + removedTask);
+//                System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+//            } else {
+//                throw new InvalidTaskNumberException("Invalid task number. Please provide a valid number from the list.");
+//            }
+//        } catch (NumberFormatException e) {
+//            throw new InvalidTaskNumberException("Invalid command format. Please enter a valid number after 'delete'.");
+//        } catch (IndexOutOfBoundsException e) {
+//            throw new InvalidTaskNumberException("Task number out of range. Please provide a valid number from the list.");
+//        }
+//    }
+//}
```

Comment:
Could I clarify what this really long chunk of code (200+) commented out is doing? Consider moving it to another file if you really need to keep a backup of past code.